libs/designsystem/src/lib/helpers/design-token-helper.ts

```diff
@@ -91,6 +100,10 @@ export class DesignTokenHelper {
   public static getElevation(z: 2 | 4 | 8): string {
     return styles.elevationsRgb[z];
   }
+
+  public static itemHeight(key: string): string {
```

Comment:
Consider strong typing (should be applied to other "styles map" methods as well):
```suggestion
  public static itemHeight(key: keyof typeof styles.itemHeights): string {
```